samples/subsys/module_hello/sample.yaml

```diff
@@ -0,0 +1,12 @@
+sample:
+  description: Module Hello sample, application made entirely from
+    subsystems and drivers
+  name: module hello
+common:
+    tags: introduction
```

Comment:
What tags: should be used here, considering this is a test case for the Zephyr document generation?